.eslintrc

```diff
@@ -123,20 +124,20 @@
     {
       "files": ["*.tsx"],
       "rules": {
-        "react/prop-types": "off"
+        "react/prop-types": "off",
+        "react/jsx-props-no-spreading": "off"
       }
     },
     {
-      "files": ["stories/**/*.{js,ts,tsx}"],
+      "files": ["src/icons/**/svgs/index.tsx"],
       "rules": {
-        "no-alert": "off"
+        "prettier/prettier": "off"
```

Comment:
The `index.tsx` file generated by `@svgr/cli` lacks a newline. Without this, the respective files will always appear to have been modified, which is annoying. 🙃